src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,28 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.response.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.dto.mapping.impl.response.UserResponseMapper;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+    private final UserService userService;
+    private final UserResponseMapper userResponseMapper;
+
+    public UserController(UserService userService, UserResponseMapper userResponseMapper) {
+        this.userService = userService;
+        this.userResponseMapper = userResponseMapper;
+    }
+
+    @GetMapping("/get-by-email")
```

Comment:
```suggestion
    @GetMapping("/by-email")
```

We don't need to write actions in the url